hydromt/nodata.py

```diff
@@ -0,0 +1,22 @@
+"""Functions to handle nodata values.""" ""
+from enum import Enum
+from logging import Logger
+
+from .exceptions import NoDataException
+
+
+class NoDataStrategy(Enum):
+    """Strategy to handle nodata values."""
+
+    RAISE = "raise"
+    IGNORE = "ignore"
```

Comment:
Perhaps a "coerce" option might be useful in some places (like STAC), but that is harder to handle generically so I'll leave it up to you whether you want to add it here.

Reply:
We can add it once we implement it :)